src/main.rs

```diff
@@ -20,6 +22,21 @@ fn main() -> Result<()> {
     }
     env_logger::init();
 
+    StdErr::deprecation_warning(&format!(
```

Comment:
If we're only using `StdErr::deprecation_warning` here, why are we also defining `StdErr::billboard`?

Reply:
Previous code defined billboard for stdout, I figured it was necessary to have both because of the code here: 
https://github.com/cloudflare/wrangler/blob/38887e9a976292e0cdfa979097b88d4cf1ad9810/src/terminal/message.rs#L49

Reply:
@jspspike / @caass pls halp

Reply:
Because `billboard` and `deprecation_warning` are both methods defined as part of a [trait](https://doc.rust-lang.org/book/ch10-02-traits.html), both of them need to be defined -- you can think of it as implementing an interface for a class in JS, and you only really want access to one of the class methods.

You can leave the billboard method unimplemented, however, and just use the `unimplemented!()` macro to tell the compiler "yea so i know i just said i would do this but i'm actually not gonna":

```rust
    fn billboard(_msg: &str) {
        unimplemented!("Can't display billboard to stderr.")
    }
```